.buildkite/pipelines/pull_request/base.yml

```diff
@@ -30,6 +30,17 @@ steps:
         - exit_status: '-1'
           limit: 3
 
+  - command: .buildkite/scripts/steps/capture_oas_snapshot.sh
```

Comment:
Mind if we source this in `.buildkite/scripts/steps/checks.sh` instead?  It'll save a few minutes by avoiding a `yarn kbn bootstrap` call

Reply:
It'll also run in the kibana-on-merge pipeline.